tests/integration_test.rs

```diff
@@ -80,8 +80,93 @@ fn identity_map() {
              excusable);
 }
 
+#[test]
+#[cfg(not(target_os = "macos"))]
+fn with_functions() {
+    // Ignored on macOS since atos doesn't support -f
+
+    let target = env::current_exe().unwrap();
+    let mut debug = target.clone();
+
+    // On macOS, symbols are kept in a separate .dSYM file
+    if cfg!(target_os = "macos") {
```

Comment:
This will do nothing, since this function isn't compiled for `target_os = "macos"`.

This seems to happen a few more times in this function, too.